routing_common/transit_types.cpp

```diff
@@ -227,14 +226,44 @@ bool ShapeId::IsValid() const
   return m_stop1Id != kInvalidStopId && m_stop2Id != kInvalidStopId;
 }
 
+// EdgeFlags --------------------------------------------------------------------------------------
+uint8_t EdgeFlags::GetFlags() const
+{
+  return BoolToUint(m_transfer) * kTransferMask +
+         BoolToUint(m_isShapeIdsEmpty) * kEmptyShapeIdsMask +
+         BoolToUint(m_isShapeIdsSingle) * kSingleShapeIdMask +
+         BoolToUint(m_isShapeIdsSame) * kShapeIdIsSameMask +
+         BoolToUint(m_isShapeIdsReversed) * kShapeIdIsReversedMask;
+}
+
+void EdgeFlags::SetFlags(uint8_t flags)
+{
+  m_transfer = GetBit(flags, kTransferMask);
+  m_isShapeIdsEmpty = GetBit(flags, kEmptyShapeIdsMask);
+  m_isShapeIdsSingle = GetBit(flags, kSingleShapeIdMask);
+  m_isShapeIdsSame = GetBit(flags, kShapeIdIsSameMask);
+  m_isShapeIdsReversed = GetBit(flags, kShapeIdIsReversedMask);
+}
+
+string DebugPrint(EdgeFlags const & f)
+{
+  std::ostringstream ss;
+  ss << "EdgeFlags [m_transfer:" << f.m_transfer;
```

Comment:
No need in "m_" in the DebugPrint messages.

Reply:
Removed